### PR TITLE
Update Avatar button and Dropdown Menu

### PR DESCRIPTION
**Pull request by azanbinzahid**
### [PROD-2274](https://openedx.atlassian.net/browse/PROD-2274)

This PR updates the implementation of user menu along with avatar images with `edx/paragon` `Dropdown` and `AvatarButton` in desktop and mobile header.

### CODE REVIEW

**Comment by awaisdar001** [comment]
@azanbinzahid can you add a little description of what this PR fixes or enhances?

File: src/support-header/DesktopHeader.jsx [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 // Local Components
+import { Dropdown, AvatarButton } from '@edx/paragon';
 import { Menu, MenuTrigger, MenuContent } from './Menu';
-import Avatar from './Avatar';
```
Comment (awaisdar001) [inline src/support-header/DesktopHeader.jsx:6]: is there usage of `Avatar.jsx` in the codebase? if not, do you think we can remove the file safely?

Comment (azanbinzahid) [reply src/support-header/DesktopHeader.jsx:6]: Yes, with this implementation we would not need `Avatar.jsx` I have removed it in recently pushed commit

**Review by DawoudSheraz — APPROVED** [review]
verify your changes on stage before deployment to production.

**Review by awaisdar001 — APPROVED** [review]

**Pull request merged**